docs/topics/proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc

```diff
@@ -25,18 +24,18 @@ $ oc new-project {project-name}
 ifndef::circuit-breaker-nodejs[]
 . Navigate to the root directory of your booster.
 
-. Use Maven to start the deployment to your {OpenShiftLocal}.
+. Use Maven to start the deployment to OpenShift.
```

Comment:
Do we want to hardcode OpenShift here? I genuinely don't know, so just asking. The same for other occurrences of this string.